types/d3-random/index.d.ts

```diff
@@ -23,18 +26,51 @@ export interface RandomNumberGenerationSource {
  */
 export interface RandomUniform extends RandomNumberGenerationSource {
     /**
-     * Returns a function for generating random numbers with a uniform distribution).
-     * The minimum allowed value of a returned number is min, and the maximum is max.
-     * If min is not specified, it defaults to 0; if max is not specified, it defaults to 1.
+     * Returns a function for generating random numbers with a uniform distribution.
+     * The minimum allowed value of a returned number is min (inclusive), and the maximum is max (exclusive).
```

Comment:
This comment looks out of date - I think `min` is gone?

Reply:
Pushed a change. "Min" "defaults" to 0, so it's still from 0 (inclusive) to max (exclusive).